internal/clients/team3/client.go

```diff
@@ -3,42 +3,56 @@ package team3
 
 import (
 	"fmt"
-	"github.com/SOMAS2020/SOMAS2020/internal/common"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
 	"log"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
 )
 
-const id = common.Team3
+const id = shared.Team3
 
 func init() {
 	common.RegisterClient(id, &client{id: id})
 }
 
 type client struct {
-	id common.ClientID
+	id shared.ClientID
 }
 
 func (c *client) Echo(s string) string {
 
-	//c.DemoEvaluation()
-	c.Logf("Echo: '%v'", s)
+	c.logf("Echo: '%v'", s)
 
 	return s
 }
 
 func (c *client) DemoEvaluation() {
-	evalResult, err := common.BasicRuleEvaluator("Kinda Complicated Rule")
+	evalResult, err := rules.BasicBooleanRuleEvaluator("Kinda Complicated Rule")
 	if err != nil {
 		panic(err.Error())
 	}
-	c.Logf("Rule Eval: %t", evalResult)
+	c.logf("Rule Eval: %t", evalResult)
 }
 
-func (c *client) GetID() common.ClientID {
+func (c *client) GetID() shared.ClientID {
```

Comment:
This should be a merge conflict... the GetID method is implemented in the BaseClient now see #45. A feature addition should not change the individual client files at all following that PR, only the BaseClient